examples/text_labels_and_annotations/text_alignment.py

```diff
@@ -1,12 +1,48 @@
 """
-===================
-Precise text layout
-===================
+==============
+Text alignment
+==============
+
+Texts are aligned relative to their anchor point depending on the properties
+``horizontalalignment`` and ``verticalalignment``.
+
+.. plot::
+
+    import matplotlib.pyplot as plt
+    import numpy as np
+
+    y = [0.22, 0.34, 0.5, 0.56, 0.77]
```

Comment:
The top line appears a bit low; did using the inverse transforms not work out?

Reply:
No the inverse does not work. See https://github.com/matplotlib/matplotlib/pull/21594#discussion_r748597231